website/blog/2024-08-08-release-1.12.md

```diff
@@ -0,0 +1,369 @@
+![remote](img/podman-desktop-release-1.12/remote.png)
+
+To set up access to a remote machine, follow the official [Podman remote-client tutorial](https://github.com/containers/podman/blob/main/docs/tutorials/remote_client.md).
+
+### macOS GPU support
+
+GPU support is now available for macOS users!
+
+This seamless setup can be enabled during Podman Machine creation by selecting the provider type:
+
+![libkrun](img/podman-desktop-release-1.12/libkrun.png)
+
+After enablement, you can test that the GPU has been supported by running a custom container with `--device /dev/dri` passed through:
+
+```sh
+$ podman run --rm -it --device /dev/dri --name gpu-info quay.io/slopezpa/fedora-vgpu vulkaninfo | grep "GPU"
+```
+
+Which will output information regarding your GPU:
+
+```sh
+
+		GPU id = 0 (Virtio-GPU Venus (Apple M1 Pro))
+		GPU id = 1 (llvmpipe (LLVM 17.0.6, 128 bits))
+GPU0:
+	deviceType        = PHYSICAL_DEVICE_TYPE_INTEGRATED_GPU
+	deviceName        = Virtio-GPU Venus (Apple M1 Pro)
+GPU1:
+```
+
+Stay tuned as we continue to improve and write documentation on GPU support.
+
+The most practical way to try these new features out is by installing [AI Lab](https://podman-desktop.io/extensions/ai-lab) and seeing the improved speed.
+
+This can be done by following the below steps:
+
+1. **Update to the latest Podman AI Lab extension** from the Extensions Catalog on Podman Desktop.
+2. Create a new Podman Machine with `libkrun` enabled (see above instructions).
+3. Enable GPU support within Podman AI Lab extension:
+
+![ai_lab](img/podman-desktop-release-1.12/ai_lab.png)
+
+4. Start an inference server / playground environment and enjoy the benefits of accelerated GPU support!
+
+### Windows GPU support
+
+Windows GPU support has been integrated into Podman for a while, however our associated extensions have now been utilizing this great feature! Such as in [AI Lab](https://podman-desktop.io/extensions/ai-lab).
+
+To showcase this awesome feature, this can be done by following the below steps:
+
+1. **Update to the latest Podman AI Lab extension** from the Extensions Catalog on Podman Desktop.
+2. Create a Podman Machine with WSL enabled.
+3. Enable GPU support within the Podman AI Lab Extension:
+
+![ai_lab](img/podman-desktop-release-1.12/ai_lab.png)
+
+4. Start an inference server / playground environment and enjoy the benefits of accelerated GPU support!
+
+### Light mode out of experimental
+
+We've made excellent progress in light mode and we have now marked it out of non-experimental! If you have not already tried it yet, this can be enabled within the interface here:
+
+![light mode](img/podman-desktop-release-1.12/light_mode.png)
+
+### Kubernetes features
+
+With this release, you can now view ConfigMaps as well as Secrets:
+
+![configmaps secrets](img/podman-desktop-release-1.12/configmaps_secrets.png)
+
+You also have the ability to now select multiple files when applying your Kubernetes YAML.
+
+---
+
+## Detailed Release Changelog
```

Comment:
I appreciate the detail, but this is pages long, much of it is minor details, and it hides things like the Community Thank You. Ideally I'd collapse the section by default, but since I don't think there is a way to do that could we move to a separate doc or at least move to the bottom?

Reply:
I agree, I think it's weird that the detailed change logs in other screens is not on the bottom anyways! i'll move it